### PR TITLE
Toolbox compatibility

### PR DESCRIPTION
**Pull request by IliaFeldgun**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -7,40 +7,25 @@ function installRPMs(){
     # sudo yum install -y epel-release
```
Comment (IliaFeldgun) [inline setup.sh:7]: Ignore this file, will need to be rewritten.

File: packer/files/python3-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+typer
```
Comment (roeigo-stratoscale) [inline packer/files/python3-requirements.txt]: should be frozen

File: packer/files/rpm-packages.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
+git
```
Comment (roeigo-stratoscale) [inline packer/files/rpm-packages.txt]: also should be frozen

File: packer/files/user-data.pkrtpl.hcl [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#cloud-config
+hostname: toolbox
+ssh_authorized_keys:
+  - ${ public_key_content }
+users:
+  - default
+#   - name: neokarm
```
Comment (roeigo-stratoscale) [inline packer/files/user-data.pkrtpl.hcl]: Can we change this to zadara?

File: packer/files/user-data.pkrtpl.hcl [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+#     gecos: Neokarm user
+#     primary_group: neokarm
+#     passwd: "$6$rounds=4096$o7ib3bkXRIFPWAId$GPV5HLt1BgBgTtxrgJ.a3pHpPJ/HEQ.jMF/G0YN90U/i7/EYcRS1q/xEZNthB4E1gSSzVjdMWWiRFJ25I/RdP1"
```
Comment (roeigo-stratoscale) [inline packer/files/user-data.pkrtpl.hcl]: replace the password with <PASSWORD>

File: symp-docker [hunk 6]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 SYMP_IMAGE="stratoscale/symp-cli:21.10.0-43fb823a"
-docker run -i --rm $SYMP_IMAGE $@
\ No newline at end of file
+docker run -e -it --rm $SYMP_IMAGE $@
```
Comment (roeigo-stratoscale) [inline symp-docker]: will -e pick up all env vars?

**Review by roeigo-stratoscale — CHANGES_REQUESTED** [review]

File: symp-docker (hunk 6 above)
Comment (IliaFeldgun) [reply symp-docker]: No, typo.

**Pull request closed without merging**